.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.idea
```

Comment:
Do you want the .idea folder to be committed in the repo? It's the folder generated by the Intellij IDEA IDE.